### DISCUSSION TITLE
CocoaRestClient

### DISCUSSION THREAD

**Discussion by AndroidKitKat** [elrumo/macOS_Big_Sur_icons_replacements#1189] [Icon request]
Very simple & handy app, would love to see it with a new icon. 

https://mmattozzi.github.io/cocoa-rest-client/

**Comment by blaichch** [comment]
Here you go. Hope you like it!

<img width="512" alt="CocoaRestClient" src="https://user-images.githubusercontent.com/12624897/120904704-dac42080-c64d-11eb-8cba-0c1c55c79466.png">
